### ISSUE TITLE
non-deterministic behavior with batch size > 1

### ISSUE THREAD

**Issue by FiorenzoParascandolo1** [IDEA-Research/GroundingDINO#288]
I noticed this strange behavior: the logits outputs of the model vary slightly with respect to different runs, resulting in non-deterministic behavior. This effect is not needed in case of batch size = 1. Do you know what the reason could be?
